contrib/go-chi/chi/option.go

```diff
@@ -17,6 +17,7 @@ type config struct {
 	serviceName   string
 	spanOpts      []ddtrace.StartSpanOption // additional span options to be applied
 	analyticsRate float64
+	statusCheck   func(statusCode int) bool
```

Comment:
I think this isn't clear enough, perhaps `isErrorStatus` would be more specific.